### PR TITLE
[Extension] Remove dependency on cros-dpsl-js

### PR DESCRIPTION
**Pull request by JessicaDJ0807**
The DPSL function calls were deprecated by a previously merged PR: "[Extension] Extend Telemetry service #44". And the dependency on the library 'cros-dpsl-js' is removed from the 'package.json' file and the 'package-lock.json' file.

BUG: b/295125739

### CODE REVIEW

**Review by ycheng627 — APPROVED** [review]
LGTM!

No need to wait for other's approval, you can go ahead and merge it.

**Review by BaiiYuan — APPROVED** [review]

**Pull request merged**
